src/equations/compressible_navier_stokes_2d.jl

```diff
@@ -326,40 +350,69 @@ The field `boundary_value_normal_flux_function` should be a function with signat
 normal heat flux at point `x` and time `t`.
 """
 struct Adiabatic{F}
-  boundary_value_normal_flux_function::F # scaled heat flux 1/T * kappa * dT/dn
+    boundary_value_normal_flux_function::F # scaled heat flux 1/T * kappa * dT/dn
 end
 
-@inline function (boundary_condition::BoundaryConditionNavierStokesWall{<:NoSlip, <:Adiabatic})(flux_inner, u_inner, normal::AbstractVector,
-                                                                                           x, t, operator_type::Gradient,
-                                                                                           equations::CompressibleNavierStokesDiffusion2D{GradientVariablesPrimitive})
-  v1, v2 = boundary_condition.boundary_condition_velocity.boundary_value_function(x, t, equations)
-  return SVector(u_inner[1], v1, v2, u_inner[4])
+@inline function (boundary_condition::BoundaryConditionNavierStokesWall{<:NoSlip,
+                                                                        <:Adiabatic})(flux_inner,
+                                                                                      u_inner,
+                                                                                      normal::AbstractVector,
+                                                                                      x, t,
+                                                                                      operator_type::Gradient,
+                                                                                      equations::CompressibleNavierStokesDiffusion2D{
+                                                                                                                                     GradientVariablesPrimitive
+                                                                                                                                     })
```

Comment:
This is a little better with respect to parameters, IMO - having multiple lines for multiple type parameters looks more organized.